R/utils.R

```diff
@@ -13,8 +13,8 @@ vlapply <- function(X, FUN, ...) {
 }
 
 
-to_json <- function(x, auto_unbox = FALSE) {
-  V8::JS(jsonlite::toJSON(x, auto_unbox = auto_unbox, digits = NA))
+to_json <- function(x, auto_unbox = FALSE, digits = NA, ...) {
+  V8::JS(jsonlite::toJSON(x, auto_unbox = auto_unbox, digits = NA, ...))
```

Comment:
Perhaps you mean to supply digits to the function?

Reply:
ah, thanks :)